### DISCUSSION TITLE
Help: Issue during ./mvnw build

### DISCUSSION THREAD

**Discussion by myersdennis94** [provectus/kafka-ui#1557] [Q&A] [answered]
I am interested in contributing to this project and getting into open source. I am trying to build the app according to the instructions in the README but my build is crashing. The console output is below, if anyone can help in any way it is greatly appreciated!!

[INFO] --- frontend-maven-plugin:1.12.0:npm (npm run gen:sources) @ kafka-ui-contract ---
[INFO] Running 'npm run gen:sources' in D:\OpenSource\kafka-ui\kafka-ui-react-app
[INFO]
[INFO] > kafka-ui@0.1.0 gen:sources D:\OpenSource\kafka-ui\kafka-ui-react-app
[INFO] > rimraf src/generated-sources && openapi-generator-cli generate
[INFO]
[INFO] Download 5.4.0 ...
[INFO] Downloaded 5.4.0
[INFO] Did set selected version to 5.4.0
[INFO] [error] Required option '-i' is missing
[INFO] npm ERR! code ELIFECYCLE
[INFO] npm ERR! errno 1
[INFO] npm ERR! kafka-ui@0.1.0 gen:sources: `rimraf src/generated-sources && openapi-generator-cli generate`
[INFO] npm ERR! Exit status 1
[INFO] npm ERR!
[INFO] npm ERR! Failed at the kafka-ui@0.1.0 gen:sources script.
[INFO] npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
[INFO]
[INFO] npm ERR! A complete log of this run can be found in:
[INFO] npm ERR!     C:\Users\Dennis\AppData\Roaming\npm-cache\_logs\2022-02-06T04_13_00_574Z-debug.log
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for kafka-ui 0.0.1-SNAPSHOT:
[INFO]
[INFO] kafka-ui ........................................... SUCCESS [  0.242 s]
[INFO] kafka-ui-contract .................................. FAILURE [05:35 min]
[INFO] kafka-ui-api ....................................... SKIPPED
[INFO] kafka-ui-e2e-checks ................................ SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  05:35 min
[INFO] Finished at: 2022-02-05T22:13:00-06:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.12.0:npm (npm run gen:sources) on project kafka-ui-contract: Failed to run task: 'npm run gen:sources' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :kafka-ui-contract

**Comment by Haarolean** [comment]
Hey, thanks for reaching out. 

We'll take a look, that seems like something we've previously encountered on windows. I'll let you know once I get a solution.

**Comment by Haarolean** [comment, marked as answer]
@myersdennis94 I was able to reproduce the issue, the command doesn't fail when executed alone but during a full maven build it does fail. I don't have a solution for now, but as a temporary solution I can recommend using WSL for Windows (https://aka.ms/wslstore). It does build fine on WSL-ish ubuntu.

I'll convert the discussion to an issue since it's a problem with an app.
